vm/src/exceptions.rs

```diff
@@ -138,6 +139,7 @@ impl ExceptionZoo {
         let import_error = create_type("ImportError", &type_type, &exception_type, &dict_type);
         let module_not_found_error =
             create_type("ModuleNotFoundError", &type_type, &import_error, &dict_type);
+        let overflow_error = create_type("OverflowError", &type_type, &exception_type, &dict_type);
```

Comment:
This should be the [`ArithmeticError`](https://docs.python.org/3/library/exceptions.html#ArithmeticError)’s subclass. I’ve seen this, since I try to add the same exception type in my PR #347 and I’ve seen you discuss it here. :)

Reply:
@silmeth Are you planning to add it in #347 ? If so I can wait until you merge and I'll rebase this based on your implementation.

Reply:
I’ve already added it alongside `OverflowError` and `ZeroDivisionError` in #347.